schedule/advance.py

```diff
@@ -73,31 +57,19 @@ def advance(did):
         lambda x: (
             x
             + [
-                did_to_deck_parameters[x[1]]["r"],
-                exponential_forgetting_curve(x[4], x[3])
-                if version[0] == 3
-                else power_forgetting_curve(x[4], x[3]),
+                DM.config_dict_for_deck_id(x[1])["desiredRetention"],
```

Comment:
This is now available via $.dr

Reply:
What is `$.dr`?

Reply:
The desired retention is stored in each card to make sorting easier, so you don't need to fetch it from the deck preset.